bifromq-apiserver/src/main/java/com/baidu/bifromq/apiserver/NonTLSServerInitializer.java

```diff
@@ -30,6 +31,7 @@ protected void initChannel(SocketChannel ch) {
         final ChannelPipeline p = ch.pipeline();
         final HttpServerCodec sourceCodec = new HttpServerCodec();
         p.addLast(sourceCodec);
+        p.addLast(new HttpObjectAggregator(1024 * 1024));
```

Comment:
it's better to make it configurable via APIServerConfig which is default to the same value as MQTTServerConfig.maxMsgByteSize